pkg/handler/task_info.go

```diff
@@ -104,7 +105,10 @@ func (t *TaskInfoHandler) cancel(c echo.Context) error {
 		return ce.NewErrorResponse(ce.HttpCodeForDaoError(err), "error canceling task", err.Error())
 	}
 	if task.OrgId == config.RedHatOrg {
-		return ce.NewErrorResponse(http.StatusBadRequest, "Cannot cancel a Red Hat Task", err.Error())
+		return ce.NewErrorResponse(http.StatusBadRequest, "error canceling task", "Cannot cancel a Red Hat Task")
+	}
+	if !slices.Contains(config.CancellableTasks, task.Typename) {
+		return ce.NewErrorResponse(http.StatusBadRequest, "error canceling task", "Can only cancel introspect or snapshot tasks")
```

Comment:
What do you think about moving this check into this method?
https://github.com/content-services/content-sources-backend/blob/main/pkg/tasks/client/client.go#L38

This way if a task is cancelled outside of the handler, the cancel is still blocked

Reply:
makes sense! will do